### DISCUSSION TITLE
Container class

### DISCUSSION THREAD

**Discussion by Naimul07** [tailwindlabs/tailwindcss#15420] [Help]
![image](https://github.com/user-attachments/assets/6e869053-5d97-4b86-8656-7a52282c3883)

Container is not working when the screen size is exactly 1024px ,1280px. as you can see in the provided image no space was given.it is a bad taste for design.

**Comment by adamwathan** [comment]
Can you share a reproduction in Tailwind Play?

**Reply by Naimul07** [reply]
https://play.tailwindcss.com/qZPU0UiPak?size=1280x720

**Reply by philipp-spiess** [reply]
The container seems to work as expected here, when the size is exactly the breakpoint, it'll be of full width but does not overflow the viewport. If you want something else, I'd suggest not using container for this.
